code/game/turfs/space/space.dm

```diff
@@ -15,8 +15,20 @@
 
 /turf/space/New()
 	. = ..()
+	if(!global_space_area)
+		global_space_area = new /area/space()
+		global_space_area.white_overlay = image(loc=global_space_area,icon='icons/turf/space.dmi',icon_state="white",layer=AREA_LAYER+0.1)
+	if(istype(loc,/area/space))
+		global_space_area.contents += src
+	else
+		var/area/A = loc
+		if(!A.white_overlay)
+			A.white_overlay = image(loc=A,icon='icons/turf/space.dmi',icon_state="white",layer=AREA_LAYER+0.1)
+			for(var/client/C in parallax_on_clients)
+				C.images |= A.white_overlay
 

```

Comment:
The images are added somewhere in hud.dm too. But this is for when an area is added while there's players on, so that when you blow a hole in the middle of the station, that hole draws correctly for people that didn't join after the hole was created.